index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <link rel="stylesheet"
+  href="https://fonts.googleapis.com/css?family=Cormorant">
+</head>
+<body>
+  <header>
+    <h1 class="title">M. Chau</h1>
+    <nav>
+      <a href=".">About</a>
+      <a href="pages/portfolio.html">Portfolio</a>
+      <a href="pages/contact.html">Contact</a>
+    </nav>
+  </header>
+  <section>
+    <h2>M. Chau is a software developer & writer & full-time dream machine.</h2>
+    <p>She is an <a href="https://adadevelopersacademy.org/">Adie</a> (Digital Cohort 18) of 2022-23. Another human in the world, is passionate about environmental justice, inclusive digital communities, and science fiction and fantasy storytelling. In her free time, she works on her own fiction, enjoys narratively and thematically rich video games, and is trying to improve her home cooking. As a developer, she is working on full-stack development (Python, Flask, HTML & CSS, JS, React). She has lived in both the U.S. and China.</p>
+  </section>
+  <footer></footer>
```

Comment:
I guess you didn't think of anything to put here, but that's fine.

(Surprisingly, the HTML validator does _not_ flag the empty element here.)